src/App.js

```diff
@@ -12,26 +12,23 @@ import CategoriesList from './redux/categories/CategoriesList';
 import Category from './redux/categories/Category';
 import NotFoundPage from './components/pages/NotFoundPage';
 
+// import { removeBook, updateBook } from './redux/books/booksSlice';
+
```

Comment:
### Optional
- [ ] It would be nice if we remove all commented piece of code, it makes our codebase looks messy and difficult to understand

Reply:
Done @addod19. All commented code has been removed.